cluster/src/test/scala/io/snappydata/tools/LeaderLauncherSpec.scala

```diff
@@ -90,7 +87,10 @@ class LeaderLauncherSpec extends WordSpec with Matchers {
 
         {
           // Stop if already any present
-          SnappyContext.stop()
+          if (SnappyContext.globalSparkContext != null &&
+              !SnappyContext.globalSparkContext.isStopped) {
+            SnappyContext.globalSparkContext.stop
+          }
```

Comment:
Since this is getting invoked in many places, why not simply have this code inside SnappyContext.stop()?